demos/capi/README.md

```diff
@@ -0,0 +1,89 @@
+# C and C++ API inference demo (C/C++)
+
+This demo creates a new docker image openvino/model_server-capi with two example applications that are using c or c++ ovms API.
+
+## Build ovms_shared.so
```

Comment:
Do not mix namings: library name is `libovms_shared.so` but the target is `ovms_shared`. There is no such thing as `ovms_shared.so`